### DISCUSSION TITLE
Home page marketing/branding

### DISCUSSION THREAD

**Discussion by joemasilotti** [joemasilotti/railsdevs.com#115] [Ideas]
I don't think the homepage hero is as powerful, or informative, as it could be.

#### railsdevs is a place for companies to find Ruby on Rails developers who are looking for freelance, contract, and full-time work.

<img width="946" alt="image" src="https://user-images.githubusercontent.com/2092156/142748553-de4505fd-c18b-46ac-856d-00359e199fb5.png">

---

In a recent response to one of my tweets, someone suggested the following ideas.

> Hiring for Rails?
> Hire Rails developers
> Hiring for a Rails project?

I like the _idea_ of these better, but I'm not sure I love the word "hire." Is that the right term to use even if you're working with someone on a contract or freelance? "Hire" to me means full-time, which is not exactly the priority of the site.

In my mind, I want railsdevs.com to be _the_ place to go to to _find_ the best Rails developers. Part-time, freelance, and even full-time (in that order). If a company, or individual, is looking for a Rails developer to help out I want them to think of railsdevs.com first.

---

Anyway, I'd love to hear what folks think on this. We can find some copy together that better explains the vision for the site!

**Comment by davegudge** [comment]
Perhaps you have defined what you're after in the explanation?

_Find the best Rails developers. Part-time, freelance, and even full-time, look no further than railsdevs.com_

**Reply by joemasilotti** [reply]
That's a good point. I just don't think that "find" is a strong enough word for what the site is trying to sell.
